### DISCUSSION TITLE
When I try to run artillery test I get this error: Cannot read properties of null (reading 'match')

### DISCUSSION THREAD

**Discussion by GustavoMoscovich** [artilleryio/artillery#2237] [Artillery]
Hi I´m trying to run artillery for first time in my proyect.
I´ve added this script in package.json file: "test-simple": "artillery quick --count 40 --num 50 'http://localhost:8080/api/simple' -o ./src/tests/performance/simple.json"

When I tried to execute I got this error...

PS C:\Users\gmoscovich\OneDrive\CODER\Programacion BACK\Desafio_03> npm run test-simple

> desafio_03@1.0.0 test-simple
> artillery quick --count 40 --num 50 'http://localhost:8080/api/simple' -o ./src/tests/performance/simple.json

    TypeError: Cannot read properties of null (reading 'match')

**Comment by bernardobridge** [comment]
Hi @GustavoMoscovich 👋 ,

I can't reproduce the error. Are you able to run it as a regular Artillery script? (`artillery run` and a YAML file).
